native/rhai_rustler/src/ast.rs

```diff
@@ -0,0 +1,7 @@
+use std::sync::Mutex;
+
+use rhai::AST;
+
+pub struct ASTResource {
+    pub ast: Mutex<AST>,
```

Comment:
probably for AST is worth using `RwLock` instead

Reply:
Done